src/utils/utils.py

```diff
@@ -368,552 +368,6 @@ def split_data(data, ratio, shuffle=1):
-        which contains the unnormalized probability for each class.
-    targets : ``torch.LongTensor``, required.
-        A ``torch.LongTensor`` of size (batch, sequence_length) which contains the
-        index of the true class for each corresponding step.
-    weights : ``torch.FloatTensor``, required.
-        A ``torch.FloatTensor`` of size (batch, sequence_length)
-    batch_average : bool, optional, (default = True).
-        A bool indicating whether the loss should be averaged across the batch,
-        or returned as a vector of losses per batch element.
-
-    Returns
-    -------
-    A torch.FloatTensor representing the cross entropy loss.
-    If ``batch_average == True``, the returned loss is a scalar.
-    If ``batch_average == False``, the returned loss is a vector of shape (batch_size,).
-
-    """
-    # shape : (batch * sequence_length, num_classes)
-    logits_flat = logits.view(-1, logits.size(-1))
-    # shape : (batch * sequence_length, num_classes)
-    log_probs_flat = torch.nn.functional.log_softmax(logits_flat)
-    # shape : (batch * max_len, 1)
-    targets_flat = targets.view(-1, 1).long()
-
-    # Contribution to the negative log likelihood only comes from the exact indices
-    # of the targets, as the target distributions are one-hot. Here we use torch.gather
-    # to extract the indices of the num_classes dimension which contribute to the loss.
-    # shape : (batch * sequence_length, 1)
-    negative_log_likelihood_flat = - torch.gather(log_probs_flat, dim=1, index=targets_flat)
-    # shape : (batch, sequence_length)
-    negative_log_likelihood = negative_log_likelihood_flat.view(*targets.size())
-    # shape : (batch, sequence_length)
-    negative_log_likelihood = negative_log_likelihood * weights.float()
-    # shape : (batch_size,)
-    per_batch_loss = negative_log_likelihood.sum(1) / (weights.sum(1).float() + 1e-13)
-
-    if batch_average:
-        num_non_empty_sequences = ((weights.sum(1) > 0).float().sum() + 1e-13)
-        return per_batch_loss.sum() / num_non_empty_sequences
-    return per_batch_loss
-
-
-def replace_masked_values(tensor: Variable, mask: Variable, replace_with: float) -> Variable:
-    """
-    Replaces all masked values in ``tensor`` with ``replace_with``.  ``mask`` must be broadcastable
-    to the same shape as ``tensor``. We require that ``tensor.dim() == mask.dim()``, as otherwise we
-    won't know which dimensions of the mask to unsqueeze.
-    """
-    # We'll build a tensor of the same shape as `tensor`, zero out masked values, then add back in
-    # the `replace_with` value.
-    if tensor.dim() != mask.dim():
-        raise ConfigurationError(
-            "tensor.dim() (%d) != mask.dim() (%d)" %
-            (tensor.dim(), mask.dim()))
-    one_minus_mask = 1.0 - mask
-    values_to_add = replace_with * one_minus_mask
-    return tensor * mask + values_to_add
-
-
-def device_mapping(cuda_device: int):
```

Comment:
This is called still in `utils.py` and is causing my experiments to crash. Can we please run tests so we're not breaking things 3 weeks before a paper deadline?

https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/utils/utils.py#L69